transforms/utils/parseSync.js

```diff
@@ -89,10 +89,16 @@ function parseSync(fileInfo) {
 				} else if (tsx) {
 					return babylon.parse(code, tsxParserOptions);
 				} else {
-					return hermesParser.parse(code, {
+					const result = hermesParser.parse(code, {
 						babel: true,
 						sourceFilename: fileInfo.path,
+						tokens: true
```

Comment:
This should set the field according to docs:
> tokens: boolean, defaults to false. If true, add all tokens to a tokens property on the root node.
-- https://www.npmjs.com/package/hermes-parser

Maybe it doesn't work when `babel: true` is also set? Babel does set the field though.